server.py

```diff
@@ -79,7 +80,17 @@ def make_video():
             logging.error(e)
             return make_response("unknown", 500)
 
-
+@app.route('/api/feedback', method = ['POST'])
+def get_feedbackvalue():
+    if request.method == 'POST':
+        text = request.values.get('feedback')
```

Comment:
建議加個判斷`text`是否為`None` ，如果是的話應該要回應 Bad Request (400)
```suggestion
        text = request.values.get('feedback')
        if text is not None:
            # Some code here
        else:
            return make_response("need feedback", 400)
```